package.json

```diff
@@ -112,8 +113,17 @@
     "minimist": "^1.2.0",
     "mkdirp": "^0.5.1",
     "mocha": "^2.4.5",
+    "node-sass": "^4.1.1",
```

Comment:
if we're not using SASS, let's not drag it in.

We can lose `catw` too I think.